contracts/test/TestStrategy.sol

```diff
@@ -85,8 +91,7 @@ contract TestStrategy is BaseStrategyInitializable {
         } else {
             // NOTE: Just in case something was stolen from this contract
             if (totalDebt > totalAssets) {
-                _loss = totalDebt.sub(totalAssets);
-                if (_loss > _amountNeeded) _loss = _amountNeeded;
+                _loss = _amountNeeded;
```

Comment:
I don't think this is correct

Reply:
In my testing I've not been able to get `_loss > _amountNeeded` to return false. I believe this line is over-defined/always true, so I set `_loss = _amountNeeded` as this is what happens when `_loss > _amountNeeded` is true.

![](https://i.imgur.com/LTnQKM4.png)

Reply:
I've reverted this commit as I was able to add a new unit test to obtain full coverage without modifying this code

![image](https://user-images.githubusercontent.com/70453354/109711510-90ba3300-7b64-11eb-9234-70a37156b105.png)
![image](https://user-images.githubusercontent.com/70453354/109711541-97e14100-7b64-11eb-858a-d9a733402062.png)